lldb/source/Expression/REPL.cpp

```diff
@@ -339,12 +339,9 @@ void REPL::IOHandlerInputComplete(IOHandler &io_handler, std::string &code) {
 
       const char *expr_prefix = nullptr;
       lldb::ValueObjectSP result_valobj_sp;
+      lldb::ExpressionResults execution_results = UserExpression::Evaluate(
+          exe_ctx, expr_options, code.c_str(), expr_prefix, result_valobj_sp);
       Status error;
```

Comment:
Can you remove this variable? If this variable is checked, that can also be removed now since it will always be empty. If it wasn't checked before, well, then I guess you can remove only this line. :)

Reply:
It's used on the very next line.